src/scenes/Root/Sidebar/SwooshIcon.tsx

```diff
@@ -0,0 +1,61 @@
+import { makeStyles, SvgIcon, SvgIconProps } from '@material-ui/core';
+import * as React from 'react';
+import { FC } from 'react';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    width: '248px',
+    height: '136px',
+  },
+}));
+
+export const SwooshIcon: FC<SvgIconProps> = ({
```

Comment:
I know it's using SvgIcon, but it's not an Icon. `Swoosh` or `SwooshBackground` or something like that.

Reply:
Changed to SwooshBackground